pkg/helpers/yaml/yaml.go

```diff
@@ -1,6 +1,27 @@
-package utils
+package yaml
```

Comment:
Also, consider moving this file to `pkg/parser/yamlhelper/yamlhelper.go`, this way:

* you don't need to import it with a different name (see below)
* it will be easier to understand what this package is about (as opposed to common names like "utils")

Reply:
Moved to `pkg/yamlhelper`